pupil/views/partials/_content.ejs

```diff
@@ -1,4 +1,4 @@
-<main id="content" class="js-content" role="main">
+<main id="content" class="js-content">
```

Comment:
Why remove `role="main"`? I think the redundancy is intended looking at other gov.uk webpages.